packages/core/src/codewhispererChat/clients/chat/v0/chat.ts

```diff
@@ -35,6 +38,28 @@ export class ChatSession {
     public setSessionID(id?: string) {
         this.sessionId = id
     }
+    public get listOfReadFiles(): string[] {
+        return this._listOfReadFiles
+    }
+    public get getFilePath(): string | undefined {
```

Comment:
This is already a getter function, `get` in the name is redundant

```suggestion
    public get filePath(): string | undefined {
```